app/assets/stylesheets/views/_taxons.scss

```diff
@@ -69,33 +69,37 @@
   margin-top: $gutter * 2;
   @include taxon-element-styles;
 
-  ol {
+  .taxon-page__grid-wrapper {
     list-style-type: none;
     padding-bottom: $gutter-half;
-    @extend %grid-row;
+    font-size: 0; // Set so li elements don't wrap unnecessarily due to white space
```

Comment:
This is presumably necessary to to the use of inline-block on the child LI elements. This feels slightly hacky - have you considered using flexbox here as used on the highlight boxes?

Reply:
I did consider using flexbox, but it felt a bit overkill here, especially as flexbox isn't fully supported by IE. This does feel a bit hacky, but to me it was the better solution over using flexbox and having to implement IE-specific CSS fixes.